### DISCUSSION TITLE
Add an icon property for the window title bar within the page class

### DISCUSSION THREAD

**Discussion by burhansvural** [flet-dev/flet#2029] [Ideas]
If the window title bar icon changing feature for Windows applications is added within the page class(exam=>window_title_bar_icon='ico64.ico'), we can better evaluate how these icons are seen at runtime.

**Comment by ndonkoHenri** [comment]
The only way to change that icon at the moment is to package your application as a standalone executable file.

**Comment by theAbdoSabbagh** [comment]
Hey, is this still not possible?

**Reply by ndonkoHenri** [reply]
Yep, it isn't possible when developing the app.

**Reply by theAbdoSabbagh** [reply]
I did some digging and found this:
```py
from flet.__pyinstaller.win_utils import update_flet_view_icon
```

I was wondering if you could help me understand how this works, since I'm discussing how it fix the issue with Nuitka not being able to change the icon of flet applications on titlebar or task manager at https://github.com/Nuitka/Nuitka/issues/2909 and thus I thought maybe if I understand how that function works, it could be used in Nuitka as well

**Reply by kacper-the-programmer** [reply]
It isn't possible because of how flet works, flet is basically a flutter client with is already build, and you can't easly change icon of already build executable
